target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -11,6 +11,13 @@ board_config_update
 board=$(ipq806x_board_name)
 
 case "$board" in
+ad7200)
+	ucidef_set_led_usbdev "usb1" "USB 1" "${board}:blue:usb_1" "2-1"
+	ucidef_set_led_usbdev "usb2" "USB 2" "${board}:blue:usb_3" "4-1"
```

Comment:
Use the new ucidef_set_led_usbport here. Have a look at https://github.com/lede-project/source/commit/dafbc94f95141655f358fdd015a552921b3a66a3.

To my knowledge usb1-port1 and usb3-port1 are used in case an usb 3.0 device is attached.